jina/logging/sse.py

```diff
@@ -1,17 +1,20 @@
 __copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-import time
+import asyncio
+from typing import Optional, List
 
 from . import default_logger
 from .. import JINA_GLOBAL, __version__
 from ..helper import yaml
 
 
-def start_sse_logger(server_config_path: str, flow_yaml: str = None):
+def start_sse_logger(server_config_path: str, identity: str, pod_identities: List[str], flow_yaml: Optional[str] = None):
```

Comment:
```suggestion
def start_sse_logger(server_config_path: str, identity: str, pod_identities: Sequence[str], flow_yaml: Optional[str] = None):
```

Reply:
THis is not anymore valid, I think you were looking at an old version?